deploy/crds/aerogear_v1alpha1_unifiedpushserver_crd.yaml

```diff
@@ -33,6 +33,15 @@ spec:
           type: object
         status:
           type: object
+          properties:
+            postgresqlReady:
+              type: boolean
+              description: 'postgresqlReady is set to boolean true if the PostgreSQL
+                deployment has at least one ready Pod'
+            unifiedpushReady:
+              type: boolean
+              description: 'unifiedpushReady is set to boolean true if the UnifiedPush
+                deployment has at least one ready Pod'
```

Comment:
Not sure how important it is to add this stuff for `status` fields. I see it is usually done for `spec` fields, but I guess it does no harm, and is maybe additional docs.

Reply:
normally speaking the validation is not required on the `status` field, mainly because it is not set by users. It is more important to provide some docs to explain what values mean in the status field.